src/NzbDrone.Core/Organizer/FileNameValidationService.cs

```diff
@@ -11,12 +11,26 @@ public interface IFilenameValidationService
         ValidationFailure ValidateStandardFilename(SampleResult sampleResult);
         ValidationFailure ValidateDailyFilename(SampleResult sampleResult);
         ValidationFailure ValidateAnimeFilename(SampleResult sampleResult);
+        ValidationFailure ValidateMovieFilename(SampleResult sampleResult);
     }
 
     public class FileNameValidationService : IFilenameValidationService
     {
         private const string ERROR_MESSAGE = "Produces invalid file names";
 
+        public ValidationFailure ValidateMovieFilename(SampleResult sampleResult)
+        {
+            var validationFailure = new ValidationFailure("MovieFormat", ERROR_MESSAGE);
+            var parsedMovieInfo = Parser.Parser.ParseMovieTitle(sampleResult.FileName);
```

Comment:
The Problem here is that movie titles without years are not going to get parsed. For now we can just check if it's not empty, the user will see how the renaming works anyways.